### PR TITLE
Complete mvp

### PR DESCRIPTION
**Pull request by themagicianking**
Added all basic styling, error handling and messages, reorganized file structure, and fixed a few minor bugs.

### CODE REVIEW

**Review by daaimah123 — APPROVED** [review]
Your PR didn't have a description or much indication about any specific things you wanted review, so I did a general walk through. Things look good to me. Great job also using tailwind.

**Comment by MBHunter88** [comment]
May I suggest to add in your description, of which specific commits could benefit from a more thorough review?

**Review by vazquezea96 — APPROVED** [review]
Good job on the refactor and your test! 

One nit thing I'll suggest for future projects is to write out the word "event" and "error" instead of just using "e". It's better for readability. 

Other than that good work! 👍

File: client/src/components/owner-components/CreateOwnerForm.jsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    e.preventDefault();
+
+    let newOwner = {
+      firstname: e.target.firstname.value,
+      lastname: e.target.lastname.value,
+      phone: e.target.phone.value,
+      email: e.target.email.value,
+    };
+
+    console.log("Request body:", newOwner);
```
Comment (MBHunter88) [inline client/src/components/owner-components/CreateOwnerForm.jsx:57]: Can remove any sensitive user info from client side console.logs

File: client/src/components/owner-components/EditOwnerForm.jsx
Comment (MBHunter88) [inline client/src/components/owner-components/EditOwnerForm.jsx:1]: Could you possibly reuse the CreateOwnerForm component to edit owners by having an editing state?

**Review by MBHunter88 — COMMENTED** [review]
This project looks fantastic! I’ve left a few comments throughout and have some additional suggestions for consideration:

Reusable Routes and Components: Consider setting up a useContext or utility file to centralize shared data and fetch requests, making it easier to maintain and re-use across different components.

Integration Testing: Your unit tests are clean and concise! To strengthen the testing suite, consider adding integration tests as well. For example, you might test what happens when a user submits the pet form or checks in a pet, ensuring the expected data flows and updates occur across components.

Overall, amazing work! These are just some optional ideas to consider, but you’ve clearly put a lot of thought and effort into this project. Great job on creating something so well-structured and functional—it really shows your skill and dedication. Looking forward to seeing more of your work!

**Comment by themagicianking** [comment]
> Your PR didn't have a description or much indication about any specific things you wanted review, so I did a general walk through. Things look good to me. Great job also using tailwind.

Thank you for taking a look! I've added a description and plan to add more detail later and I also plan to add better descriptions to subsequent pull requests.

**Comment by themagicianking** [comment]
> Good job on the refactor and your test!
> 
> One nit thing I'll suggest for future projects is to write out the word "event" and "error" instead of just using "e". It's better for readability.
> 
> Other than that good work! 👍

@vazquezea96 Thanks so much for your review! I've added this enhancement to the repo's issues. :)

File: client/src/components/owner-components/EditOwnerForm.jsx
Comment (themagicianking) [reply client/src/components/owner-components/EditOwnerForm.jsx:1]: This is such a good idea! I think I can definitely do that; I've added it to my issues as an enhancement :)

File: client/src/components/owner-components/CreateOwnerForm.jsx (hunk 1 above)
Comment (themagicianking) [reply client/src/components/owner-components/CreateOwnerForm.jsx]: Resolved! Thanks :)

**Comment by themagicianking** [comment]
@MBHunter88 Thank you so much for such a thoughtful and thorough review! I plan to go through my own list of enhancements and yours as well and add all of those to my issues to be refactored. This is amazing feedback!

**Pull request merged**